mlx_lm/utils.py

```diff
@@ -454,8 +466,9 @@ def quantize_model(
     Args:
         model (nn.Module): The model to be quantized.
         config (dict): Model configuration.
-        q_group_size (int): Group size for quantization.
-        q_bits (int): Bits per weight for quantization.
+        group_size (int): Group size for quantization.
```

Comment:
The variable has gotten renamed here, but other places like line 484 and 491 have a `q_` prefix.